src/test/rbd_mirror/test_ImageReplayer.cc

```diff
@@ -619,7 +619,7 @@ TEST_F(TestImageReplayerJournal, BootstrapErrorNoJournal)
   this->create_replayer();
   C_SaferCond cond;
   this->m_replayer->start(&cond);
-  ASSERT_EQ(-ENOENT, cond.wait());
+  ASSERT_EQ(0, cond.wait());
```

Comment:
The replayer now returns 0 for these case because this create a TrashMoveRequest now and finish with 0.

Reply:
nit: Then may be rename the test name from `BootstrapErrorNoJournal` to `BootstrapNoJournal`, as there is no error now?

And the same for  `BootstrapErrorMirrorDisabled` below.